src/schema/sale/index.js

```diff
@@ -101,6 +101,39 @@ export const SaleType = new GraphQLObjectType({
           return fetch.then(invert).then(exclude(options.exclude, "id"))
         },
       },
+      artworksConnection: {
+        type: artworkConnection,
+        description: "Returns a connection of artworks for a sale.",
+        args: pageable({
+          exclude: {
+            type: new GraphQLList(GraphQLString),
+            description:
+              "List of artwork IDs to exclude from the response (irrespective of size)",
+          },
+        }),
+        resolve: (
+          { eligible_sale_artworks_count, id },
+          options,
+          _request,
+          { rootValue: { accessToken, saleArtworksLoader } }
+        ) => {
+          const { page, size, offset } = convertConnectionArgsToGravityArgs(
+            options
+          )
+          const gravityArgs = { page, size }
+          const invert = saleArtworks => map(saleArtworks, "artwork")
+          return saleArtworksLoader(id, options)
```

Comment:
I think you mean to pass `gravityArgs` (instead of the Relay/input `options`).

Reply:
It's likely that currently this 'works' in that, by default, gravity will return page 1 and size 10 (but if you switch away from `first: 10` to another combo you might notice the issue).